patroni/config.py

```diff
@@ -265,7 +265,7 @@ def _set_section_values(section, params):
                 if value:
                     ret[section][param] = value
 
-        _set_section_values('restapi', ['listen', 'connect_address', 'certfile', 'keyfile', 'cafile', 'verify_client',
+        _set_section_values('restapi', ['listen', 'connect_address', 'certfile', 'keyfile', 'cafile', 'ciphers', 'verify_client',
```

Comment:
Flake8 complains: `patroni/config.py:268:121: E501 line too long (129 > 120 characters)`.

You need to move ` 'verify_client',` to the next line.